doc/etc-and-usr-etc.md

```diff
@@ -27,6 +27,24 @@ When it comes to reading or writing configuration files, YaST uses mainly two di
 * The [good old agents](https://github.com/yast/yast-core/), which are spread through all YaST
   codebase (search for `.scr`) files.
 
+It means that we need to adapt CFA classes and agents to the new scenario. The next section proposes
+a simple solution which we have just implemented to handle modifications to `sysctl` settings and,
+the last one, proposes a complex but more general solution.
+
+## A Simple Solution for `sysctl.conf`
+
+In a nutshell, jsc#SLE-9077 states that `/etc/sysctl.conf` should not be modified.  So if you want
+to modify any `sysctl` setting, you should drop a file in `/etc/sysctl.d` containing the new values.
+
+As a first step, we have added a {Yast2::CFA::Sysctl} class which offers an API to sysctl settings.
+This new class uses `/etc/sysctl.d/50-yast.conf` instead of `/etc/sysctl.conf` to write the configuration.
+Moreover, it removes known keys from the original `/etc/sysctl.conf` to avoid confusion.
+
+The main limitation of this approach is that YaST will only read a set of known keys but, as it uses
```

Comment:
I do not get why it is limitation. It is in fact whole idea of CFA to read/modify only parts that yast need.